tests/functional/backend/corpora/common.py

```diff
@@ -0,0 +1,104 @@
+import unittest
+import os
+from backend.corpora.common.corpora_config import CorporaAuthConfig
+import requests
+import base64
+import json
+import time
+
+API_URL = {
+    "prod": "https://api.cellxgene.cziscience.com",
+    "staging": "https://api.cellxgene.staging.single-cell.czi.technology",
+    "dev": "https://api.cellxgene.dev.single-cell.czi.technology",
+    "test": "http://localhost:5000",
+}
+
+AUDIENCE = {
+    "prod": "cellxgene.cziscience.com/",
+    "staging": "cellxgene.staging.single-cell.czi.technology/",
+    "test": "cellxgene.dev.single-cell.czi.technology/",
+    "dev": "cellxgene.dev.single-cell.czi.technology/",
+}
+
+
+class BaseFunctionalTestCase(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.deployment_stage = os.environ["DEPLOYMENT_STAGE"]
+
+    def setUp(self):
+        super().setUp()
+        self.get_auth_token()
+        self.api = API_URL.get(self.deployment_stage)
+        self.test_collection_id = "005d611a-14d5-4fbf-846e-571a1f874f70"
+        self.test_file_id = "7c93775542b056e048aa474535b8e5c2"
+        self.bad_collection_id = "DNE"
+        self.bad_file_id = "DNE"
+
+    @classmethod
+    def get_auth_token(cls):
+        config = CorporaAuthConfig()
+
+        response = requests.post(
+            "https://czi-cellxgene-dev.us.auth0.com/oauth/token",
+            headers={"content-type": "application/x-www-form-urlencoded"},
+            data=dict(
+                grant_type="password",
+                username=config.test_account_username,
+                password=config.test_account_password,
+                audience=AUDIENCE.get(cls.deployment_stage),
+                scope="openid profile email offline",
+                client_id=config.client_id,
+                client_secret=config.client_secret,
+            ),
+        )
+
+        id_token = response.json()["id_token"]
+        token = {"id_token": id_token}
+        cls.cookie = base64.b64encode(json.dumps(dict(token)).encode("utf-8")).decode()
```

Comment:
Why do we need `dict(token)` here when `token` is already a dict?

Reply:
Good question. This isn't new code, but worth a fix.